duckduckgo/duckduckgo-privacy-extension
# Fix broken unit tests

**Reviewer:** @dharb 



## Description:

Fixed all the broken unit test and added a basic tracker blocking test. 
Fixed linter errors

## Steps to test this PR:
1. All tests should be passing
2. Everything else should work as expected. Try some sites and make sure trackers are blocked and popup looks okay. 

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
